src/problem-analysis/presentation.tex

```diff
@@ -0,0 +1,22 @@
+% textidote: ignore begin
+\section{Presentation of the problem}\label{sec:presentation-of-the-problem}
+% textidote: ignore end
+
+Several shortcomings from NOVA have been documented in the transcript below.
+The most noticeable is the issues regarding the~\acrshort{epos} insights and how they are not elaborate enough in its
+current state.
+
+\blockquote[Nancy, owner of NOVA]{As NOVA is a relatively new business,
+    a lot of minor as well as major issues have to be solved.
+    We are lacking~\acrshort{bi} both because we are a new business,
+    but also because our current~\acrshort{epos} does not allow for sufficient insights into our data.
```

Comment:
As this is a quote, I don't think that acronyms sound natural. Likewise epos hasn't been introduced yet. Either replace it with plaintext or `\acrlong`.

Reply:
Hmm i see your point. However it is a paraphrase and not a quote, why i personally think it is alright. It is not too clear though that it is a paraphrase, so I will change that